src/backend/commands/player/troll/close_doors.cpp

```diff
@@ -0,0 +1,30 @@
+#include "backend/player_command.hpp"
+#include "natives.hpp"
+#include "pointers.hpp"
+#include "util/teleport.hpp"
+
+namespace big
+{
+	class close_doors : player_command
+	{
+		using player_command::player_command;
+
+		virtual void execute(player_ptr player, const std::vector<std::uint64_t>& _args, const std::shared_ptr<command_context> ctx)
+		{
+		    Ped ped = PLAYER::GET_PLAYER_PED_SCRIPT_INDEX(player->id());
+		    if (!PED::IS_PED_IN_ANY_VEHICLE(ped, true))
+		    	g_notification_service->push_warning("Toxic", "Target player is not in a vehicle.");
+		    else {
+		    	Vehicle vehicle = PED::GET_VEHICLE_PED_IS_USING(ped);
+
+		    	if (entity::take_control_of(vehicle))
+		    		VEHICLE::SET_VEHICLE_DOORS_SHUT(vehicle, false);
+		    	else {
+                    g_notification_service->push_warning("Toxic", "Failed to take control of player vehicle.");
+                }
+            }
+        }
+	};
```

Comment:
Weird indentation
![image](https://user-images.githubusercontent.com/24669514/214397594-486e08e3-6e15-4a27-967a-4317db98c983.png)


Also add curly brackets next to your if statement
```cpp
if (!PED::IS_PED_IN_ANY_VEHICLE(ped, true))
{
    g_notification_service->push_warning("Toxic", "Target player is not in a vehicle.");
}
```